### PR TITLE
Add missing fields to SecurityAdvisoryEvent and rename others

### PR DESCRIPTION
**Pull request by nikpivkin**
Add missing fields to [SecurityAdvisoryEvent](https://docs.github.com/en/webhooks-and-events/webhooks/webhook-events-and-payloads#security_advisory)

### CODE REVIEW

File: github/event_types.go
Comment (gmlewis) [inline github/event_types.go:1594]: I can't do a full review right now, but I noticed that `cvss` is singular (not plural) and is an acronym for "CVS Score" so its capitalization should be "CVSS" . 

The same is true for `AdvisoryCVSS`.

**Comment by gmlewis** [comment]
Ah!  I see that you are fixing some legacy naming mistakes.  Thank you, @nikpivkin !
Marking this PR as "breaking API changes" out of necessity.

**Review by gmlewis — APPROVED** [review]
Thank you, @nikpivkin !
LGTM.

Awaiting second LGTM+Approval from any other contributor to this repo before merging.

**Review by WillAbides — APPROVED** [review]

**Comment by gmlewis** [comment]
Thank you, @WillAbides !
Merging.

**Pull request merged**
